queries/sql/highlights.scm

```diff
@@ -1,9 +1,8 @@
-(function_call
-  (invocation
-    name: (identifier) @function.call
-    parameter: [(field)]? @parameter))
+(invocation
+  name: (identifier) @function.call
+  parameter: [(field)]? @parameter)
```

Comment:
why the extra `[`/`]`?

Reply:
In short: I'm a syntax noob and thought it was required